gradle.properties

```diff
@@ -1,3 +1,3 @@
-version=0.11.1-rc1
+version=v0.11.1-rc1
```

Comment:
Please use v0.10.3-rc5 as next publis release is v0.10.

Reply:
Updated.